packages/good-design/package.json

```diff
@@ -65,6 +64,7 @@
     "@babel/core": "^7.18.10",
     "@babel/runtime": "^7.18.9",
     "@gooddollar/goodprotocol": "1.0.29-beta.5",
+    "@gooddollar/web3sdk-v2": "workspace:^",
```

Comment:
we cannot merge with this, needs to be updated to the upcoming version number

Reply:
Should be as is done now on master
Move to devDependencies and set to 'latest'